src/api/app/views/webui/user/_comments.html.haml

```diff
@@ -0,0 +1,4 @@
+.card.mb-3
+  .card-body
+    - comments.each do |comment|
+      = render BsRequestCommentComponent.new(comment: comment, level: 1, commentable: comment.commentable, nodecoration: true)
```

Comment:
I wanted to refactor this component and use only the part that renders the comment bubble, but it's rapidly becoming a rabbit hole, so I'll do it in a follow up

Reply:
We really need to do something about this partial. This one place is used for all the comments (bs_request/request timeline/project/packages). Some of these places have different use-case and can't keep using the same partial everywhere

Reply:
Let's see if I can do something with https://github.com/openSUSE/open-build-service/pull/15542#pullrequestreview-1853186851